internal/configs/hcl2shim/mock_value_composer.go

```diff
@@ -310,7 +312,9 @@ func (mvc mockValueComposer) getMockValueByType(t cty.Type) (cty.Value, bool) {
 		objVals := make(map[string]cty.Value)
 
 		// populate the object with mock values
-		for k, at := range t.AttributeTypes() {
+		for _, kv := range mapToSortedSlice(t.AttributeTypes()) {
```

Comment:
Can you add a comment to explain why we want to iterate over the attribute types in a sorted manner please?